src/marqo/core/unstructured_vespa_index/unstructured_vespa_index.py

```diff
@@ -90,10 +86,15 @@ def _to_vespa_tensor_query(self, marqo_query: MarqoTensorQuery) -> Dict[str, Any
         return query
 
     @staticmethod
-    def _get_tensor_search_term(marqo_query: MarqoQuery) -> str:
+    def _get_tensor_search_term(marqo_query: MarqoTensorQuery) -> str:
         field_to_search = unstructured_common.VESPA_DOC_EMBEDDINGS
 
-        return (f"({{targetHits:{marqo_query.limit}, approximate:{str(marqo_query.approximate)}}}"
+        if marqo_query.ef_search is not None:
+            additional_hits = f' hnsw.exploreAdditionalHits:{marqo_query.ef_search - marqo_query.limit}'
```

Comment:
why is limit involved in the efsearch calculation?

Reply:
Vespa has `hnsw.exploreAdditionalHits` such that `efSearch=targetHits+additionalHits` so we have to calculate the difference here

Reply:
do we need validation? what happens if this limit is negative?